### PR TITLE
when vagrant up failed, exit 1

### PR DESCRIPTION
**Pull request by changev**
Especially in order to exit when vagrant up fails for saving time
@jlongever @panpan0000 @RackHD/corecommitters

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -102,6 +102,10 @@ vagrantUp() {
   cd ${WORKSPACE}/RackHD/example
   cp -rf ${WORKSPACE}/build-config/vagrant/* .
   CONFIG_DIR=${CONFIG_PATH} WORKSPACE=${WORKSPACE} REPO_NAME=${REPO_NAME} vagrant up --provision
+  if [ $? == 1 ]; then
```
Comment (yyscamper) [inline test.sh]: Since the vagrant up can fail due to a lot reason, so it's better change to `$? -ne 0`

**Comment by benbp** [comment]
PR title a little misleading since you're not adding `set -e` ?

**Comment by panpan0000** [comment]
:+1:

**Comment by changev** [comment]
@benbp I update the code but forgot update PR title, now fixed

**Comment by panpan0000** [comment]
test this please

**Review by yyscamper — APPROVED** [review]
+1

**Pull request merged**